.gitlab-ci.yml

```diff
@@ -142,7 +142,7 @@ variables:
   S3_DSD6_URI: s3://dsd6-staging
   RELEASE_VERSION_6: nightly
   RELEASE_VERSION_7: nightly-a7
-  DATADOG_AGENT_BUILDIMAGES: v12605723-c5447ff
+  DATADOG_AGENT_BUILDIMAGES: v13244204-27cbb54
```

Comment:
Note: this is a branch image, will be updated once https://github.com/DataDog/datadog-agent-buildimages/pull/303 is merged.

Reply:
This was updated to the main tag in the last commit